### PR TITLE
Add hero path wave animation

### PR DESCRIPTION
**Pull request by taniaprocha**

### CODE REVIEW

**Comment by filipebfernandes** [comment]
Just checking, do we want the `growth` word to start cut out?
Screenshot 2020-05-25 at 09 40 55

**Comment by taniaprocha** [comment]
@filipebfernandes good catch, thanks!

File: src/sections/CareersHero/index.js [hunk 1]
```diff
@@ -1,47 +1,116 @@
...
+            <Typography color="oxford-blue" weight="bold" variant="h1">
+              growth
+            </Typography>
+          </div>
+          <div styleName="second-text">
+            <svg
+              id="text-container"
+              ref={textContainerRef}
+              viewBox="0 0 1337 123"
+              xmlns="http://www.w3.org/2000/svg"
```
Comment (filipebfernandes) [inline src/sections/CareersHero/index.js:62]: do we need this?

File: src/sections/CareersHero/index.js [hunk 2]
```diff
@@ -1,47 +1,116 @@
...
+      : null;
+
+    if (path) {
+      const pathLength = path.getTotalLength();
+      const initialPosition = 445;
+
+      const rect = textContainer.getBoundingClientRect();
+      const scrollPercent = rect.y / 600;
+
+      updateTextPathOffset(scrollPercent * pathLength, initialPosition);
```
Comment (filipebfernandes) [inline src/sections/CareersHero/index.js:30]: if initialPosition is always 445 we don't need to create the variable and pass as parameter. It should exist inside `updateTextPathOffset`

Comment (taniaprocha) [reply src/sections/CareersHero/index.js]: True 👍

File: src/sections/CareersHero/index.js [hunk 3]
```diff
@@ -1,47 +1,116 @@
...
+    }
+  }, [scrollPosition]);
+
+  function updateTextPathOffset(offset) {
+    const initialPosition = 445;
+
+    if (scrollPosition < 250) {
+      textPath.setAttribute("startOffset", offset - initialPosition);
+    } else {
+      textPath.setAttribute("startOffset", 156);
```
Comment (filipebfernandes) [inline src/sections/CareersHero/index.js:38]: 156 could also be a var to be clearer

Comment (taniaprocha) [reply src/sections/CareersHero/index.js]: ✔️

**Pull request merged**